build.gradle.kts

```diff
@@ -0,0 +1,145 @@
+}
+version = ver + versuffix
+
+allprojects {
+    group = "com.plotsquared"
+    version = rootProject.version
+
+    repositories {
+        mavenCentral()
+        jcenter()
+
+        maven {
+            name = "Sonatype OSS"
+            url = uri("https://oss.sonatype.org/content/repositories/snapshots/")
+        }
+
+        maven {
+            name = "Jitpack"
+            url = uri("https://jitpack.io")
+        }
+
+        maven {
+            name = "IntellectualSites Repository"
+            url = uri("https://mvn.intellectualsites.com/content/repositories/snapshots")
+        }
+
+        maven {
+            name = "EngineHub Repository"
+            url = uri("https://maven.enginehub.org/repo/")
+        }
+    }
+}
+
+subprojects {
+    apply {
+        plugin<JavaPlugin>()
+        plugin<JavaLibraryPlugin>()
+        plugin<MavenPublishPlugin>()
+        plugin<ShadowPlugin>()
+        plugin<GrgitPlugin>()
+        plugin<Licenser>()
+
+        plugin<EclipsePlugin>()
+        plugin<IdeaPlugin>()
+    }
+
+    dependencies {
+        api("com.sk89q.worldedit:worldedit-core:7.2.0-SNAPSHOT") {
+            exclude(group = "bukkit-classloader-check")
+            exclude(group = "mockito-core")
+            exclude(group = "dummypermscompat")
+        }
+    }
+
+    tasks {
+        named<Delete>("clean") {
+            doFirst {
+                delete("../target")
+            }
+        }
+    }
+}
+
+allprojects {
+    dependencies {
+        // Tests
+        testImplementation("junit:junit:4.13")
+    }
+
+    configure<JavaPluginConvention> {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = sourceCompatibility
+    }
+
+    configure<LicenseExtension> {
+        header = rootProject.file("HEADER.txt")
+        include("**/*.java")
+        newLine = false
+    }
+
+    java {
+        withSourcesJar()
+        withJavadocJar()
+    }
+
+    tasks {
+        compileJava {
+            options.compilerArgs.addAll(arrayOf("-Xmaxerrs", "1000"))
+            options.compilerArgs.add("-Xlint:all")
+            for (disabledLint in arrayOf("processing", "path", "fallthrough", "serial"))
+                options.compilerArgs.add("-Xlint:$disabledLint")
+            options.isDeprecation = true
+            options.encoding = "UTF-8"
+        }
+
+        javadoc {
+            val opt = options as StandardJavadocDocletOptions
+            opt.addStringOption("Xdoclint:none", "-quiet")
+            opt.tags(
+                "apiNote:a:API Note:",
+                "implSpec:a:Implementation Requirements:",
+                "implNote:a:Implementation Note:"
+            )
+        }
+
+        shadowJar {
+            this.archiveClassifier.set(null as String?)
+            this.archiveFileName.set("${project.name}-${project.version}.${this.archiveExtension.getOrElse("jar")}")
+            this.destinationDirectory.set(rootProject.tasks.shadowJar.get().destinationDirectory.get())
```

Comment:
This doesn't set the folder to `/target/` which we are deleting on a previous step.